server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const cors = require('cors');
+const blogPostsRouter = require('./routes/blogPostsRoutes');
+const server = express();
+server.use(express.json());
+server.use(cors());
+
+//Routes
+server.use('/api/posts', blogPostsRouter);
+
+//Default send
+server.get('/', async (req, res) => {
+	res.send(`
+      <h2>Sanity check! Woot!</h2>
+    `);
+});
+
+module.exports = server;
```

Comment:
Server is using cors and working, sanity check addition along with routes make for good server code :heavy_check_mark: